SeanJones_SegueProblem/SeanJones_SegueProblem/SecondPageViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  SecondPageViewController.swift
+//  SeanJones_SegueProblem
+//
+//  Created by Sean Jones on 10/12/21.
+//
+
+import UIKit
+
+class SecondPageViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+   
+    
+    //MARK: - PROPERTIES
+    
+    var textInputs = [String]()
+    
+    //MARK: - OUTLETS
+    
+    @IBOutlet weak var inputsTableView: UITableView!
+    
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        inputsTableView.delegate = self
```

Comment:
don't need tableView.delegate if we aren't using any tableView delegate function
dataSource functions are good